src/main/kotlin/no/nav/familie/ef/sak/config/DatabaseConfiguration.kt

```diff
@@ -84,7 +79,11 @@ class DatabaseConfiguration : AbstractJdbcConfiguration() {
                                             FilTilBytearrayConverter(),
                                             BytearrayTilFilConverter(),
                                             PGobjectTilGrunnlagsdataConverter(),
-                                            GrunnlagsdataTilPGobjectConverter()
+                                            GrunnlagsdataTilPGobjectConverter(),
+                                            PGobjectTilVedtaksperioder(),
+                                            VedtaksperiodeTilPGobjectConverter(),
+                                            PGobjectTilInntektsperiode(),
+                                            InntektsperiodeTilPGobjectConverter()
```

Comment:
Burde vi hatt en generisk wrapper for lister? @blommish

Reply:
Hadde vært fint, litt usikker på om det virker :/

Reply:
Det "finnes" innebygd (evt. injectes fra et magisk sted via-via-via noe spring-oppsett), men den er i praksis defekt. Husker ikke helt hva som var tullete med den, men brukte 5 timer på å krangle med defaulten før jeg ga opp og laget wrapper selv. 

Kunne laget en `ListeWrapperDb` som løser dette, men det er kanskje litt stygt det også?